sound/soc/sof/intel/tgl.c

```diff
@@ -25,7 +25,7 @@ const struct snd_sof_dsp_ops sof_tgl_ops = {
 	/* probe/remove/shutdown */
 	.probe		= hda_dsp_probe,
 	.remove		= hda_dsp_remove,
-	.shutdown	= hda_dsp_remove,
+	.shutdown	= hda_dsp_shutdown,
```

Comment:
Can you squash this patch and the previous one together, so that this can be a single fix that can be applied on stable kernels?

Also you used the wrong Fixes tag, you have to use the SHA1 on the upstream commit, not the one of the sof-dev branch.

This should be 44a4cfad8d78efcda9ec0dd97ceea38d8b602f24

Reply:
@plbossart Thanks for pointing it out. I will update the tag and squash them.